styles.css

```diff
@@ -6,419 +6,1824 @@
+    margin-top: 30px;
+    width: 684px;
+    justify-content: flex-start;
+    padding-bottom: 1.5%;
+    padding-left: 2.5%;
+    margin-bottom: 30px;
+    color: #979493;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+  }
+
+  textarea {
+    width: 684px;
+    margin-right: 20%;
+    margin-bottom: 24px;
+    background: #fbf8f7;
+    height: 114px;
+    padding: 2.5%;
+    font-style: italic;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    border: none;
+    color: #3c3a39;
+  }
+
+  button.btz {
+    margin-right: 80%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 122px;
+    height: 48px;
+    color: #fff;
+    background: #f55800;
+    margin-bottom: 150px;
+    border: none;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+  }
+
+  .btz:enabled {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 130px;
+    height: 40px;
+    left: 100px;
+    top: calc(50% - 48px / 2 + 46.5px);
+    background: #f55800;
+    justify-content: center;
+  }
+
+  .btz:disabled {
+    width: 69px;
+    height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #5b5957;
+  }
+
+  .btz:hover {
+    color: #f55800;
+    background: #fff5e1;
+    border: 1px solid #f55800;
+  }
+
+  .btz:active {
+    width: 69px;
+    height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #f4f1f0;
+  }
+
+  .btn-settings:hover {
+    color: #f55800;
+    background: #fff5e1;
+    border: 1px solid #f55800;
+  }
+
+  .resume:hover {
+    color: #f55800;
+    background: #fff5e1;
+    border: 1px solid #f55800;
+  }
+
+  footer {
+    display: flex;
+    justify-content: center;
+  }
+
+  .border {
+    border: 1px solid #dfe1e6;
+    margin-bottom: 8px;
+  }
+
+  #foots {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .footer-icons {
+    display: flex;
+    gap: 24px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+  }
+
+  .footer-icons li img:hover {
+    width: 15px;
+    height: 15px;
+  }
+
+  .desktpopupcontainer {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    z-index: 10;
+    background-color: rgba(0, 0, 0, 0.85);
+    display: none;
+    backdrop-filter: blur (30px);
+  }
+
+  .expanded {
+    display: block;
+  }
```

Comment:
- [ ] It seems like there's an empty space at the bottom of the popup please make sure to remove it so it looks more like the Figma design.✔
- Here's how it looks:
   ![Inked5Hs2CIDP2l](https://user-images.githubusercontent.com/104968236/232014068-a5039e18-340a-44ec-893a-fea527c4cb4d.jpg)